### PR TITLE
Enable SDL policheck tool

### PR DESCRIPTION
**Pull request by JoeRobich**
This change enables policheck for official builds. ([Test Run](https://dev.azure.com/dnceng/internal/_build/results?buildId=2483276&view=logs&j=bc38e8b8-e027-53cb-48e7-2adbd1070eca&t=a2a8b996-32f3-5822-d162-1e94917482b8))

### CODE REVIEW

File: .config/guardian/TSAConfig.gdntsa [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "codebaseName": "VSCode-CSharp-GitHub",
+    "notificationAliases": [
+        "mlinfraswat@microsoft.com"
+    ],
+    "codebaseAdmins": [
+        "REDMOND\\jaredpar",
+        "REDMOND\\arkalyan"
+    ],
+    "instanceUrl": "https://devdiv.visualstudio.com",
+    "projectName": "DevDiv",
```
Comment (dibarbet) [inline .config/guardian/TSAConfig.gdntsa:11]: Is this correct?  Would have thought it would be dnceng

File: .config/guardian/TSAConfig.gdntsa [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "codebaseName": "VSCode-CSharp-GitHub",
+    "notificationAliases": [
+        "mlinfraswat@microsoft.com"
+    ],
+    "codebaseAdmins": [
+        "REDMOND\\jaredpar",
```
Comment (dibarbet) [inline .config/guardian/TSAConfig.gdntsa:7]: Probably replace Jared with Phil

Comment (dibarbet) [reply .config/guardian/TSAConfig.gdntsa:7]: Unless this is automatically generated

File: .config/guardian/TSAConfig.gdntsa (hunk 1 above)
Comment (JoeRobich) [reply .config/guardian/TSAConfig.gdntsa:11]: Our issues are tracked in DevDiv

File: .config/guardian/TSAConfig.gdntsa (hunk 2 above)
Comment (JoeRobich) [reply .config/guardian/TSAConfig.gdntsa:7]: nah, just copied from Roslyn. Will updated.

Comment (JoeRobich) [inline .config/guardian/TSAConfig.gdntsa:7]:
```suggestion
        "REDMOND\\phillipa",
```

Comment (JoeRobich) [reply .config/guardian/TSAConfig.gdntsa]: Updated.

**Review by dibarbet — APPROVED** [review]

**Pull request merged**